src/server/scripts/EasternKingdoms/BlackrockMountain/BlackrockDepths/blackrock_depths.cpp

```diff
@@ -1208,10 +586,4 @@ void AddSC_blackrock_depths()
     new npc_phalanx();
     new npc_lokhtos_darkbargainer();
     new npc_rocknot();
-    // Fix us
```

Comment:
1. Are these NPC's scripted somewhere else?
2. If no, then how broken are these scripts? Unnecessary to remove scripts if they are only slightly broken and not implemented elsewhere. If that were the case it would be better to uncomment them so that the NPC's actually have a script and see over if it is correct to an extent

Reply:
The truth is that I do not know what function those lines fulfill, but I consider that if they are commented, they should not be in the emulator. It's a subjective argument, it depends on who's looking at it. I don't have enough information, so I made the pull request, to know what the rest of the community thinks about it.

Reply:
The idea was to check the gossip, and on the way, I found that script that had 611 commented lines and it called my attention. Honestly if they are part of a script, I think they should be somewhere else if they are not used. Out of 1200 lines that the whole file has, 600 are comments.

Reply:
If those scripts are unfinished, then we should finish them last, but leaving them in comment form, I think is not the best way. And if they were moved to the database, through SmartAI, they should not be commented inside the script. It's a subjective assessment.